helm/fmtok8s-conference-chart/Chart.yaml

```diff
@@ -29,3 +29,5 @@ dependencies:
 - name: fmtok8s-email-service
   repository: https://salaboy.github.io/helm/
   version: v0.1.0
+- name: bitnami/kube-prometheus
+  repository: https://charts.bitnami.com/bitnami
```

Comment:
@mcruzdev we need a conditional here..

Reply:
like this: 
```
condition: prometheus.enabled
```

Reply:
@salaboy done, thank you haha!

I wanted to do it